### PR TITLE
Created HTML page with form PR

### PR DESCRIPTION
**Pull request by Ostafii4uk**
- [DEMO LINK](https://Ostafii4uk.github.io/layout_html-form/)
- [TEST REPORT LINK](https://Ostafii4uk.github.io/layout_html-form/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -11,7 +11,171 @@
...
+              type="date"
+              name="fullDateOfBirthday"
+            >
+          </label>
+
+          <label>
+            I accept the term of the agreement
+            <input
+              type="checkbox"
+              name="acceptTheTermOfTheAgreement"
```
Comment (solaryasha) [inline src/index.html:64]: Okay :)

File: src/style.css [hunk 2]
```diff
@@ -1 +1,24 @@
...
+}
+
+label {
+  display: block;
+  margin-top: 10px;
+}
+
+label:first-child {
+  margin-top: 0;
+}
```
Comment (solaryasha) [inline src/style.css:16]: Don't style tags, use class selectors instead

File: src/index.html [hunk 3]
```diff
@@ -11,7 +11,171 @@
...
+        </label>
+
+        <label>
+          How do you rate our work?
+          <input
+          type="range"
+          max="100"
+          min="0"
+          value="0"
+          >
```
Comment (solaryasha) [inline src/index.html:148]:
```suggestion
          <input
            type="range"
            max="100"
            min="0"
            value="0"
          >
```

File: src/index.html [hunk 4]
```diff
@@ -11,7 +11,171 @@
...
+      </fieldset>
+
+      <fieldset>
+        <legend>Additional info:</legend>
+
+        <div>
+          <label>
+            Comments:
+            <textarea
+              name="Comments"
```
Comment (solaryasha) [inline src/index.html:159]:
```suggestion
              name="comments"
```

File: src/index.html [hunk 5]
```diff
@@ -11,7 +11,171 @@
...
+              autocomplete="off"
+            ></textarea>
+          </label>
+
+          <label>
+            Would you recommend us?
+            <select name="RecomendUs" required>
```
Comment (solaryasha) [inline src/index.html:166]:
```suggestion
            <select name="recomendUs" required>
```

**Review by solaryasha — CHANGES_REQUESTED** [review]
Cool start, let's fix some comments

**Review by solaryasha — APPROVED** [review]
Nice ! 🚀

**Comment by Ostafii4uk** [comment]
Approved

**Pull request closed without merging**